### PR TITLE
Add deploy_gallery linux try builder config

### PR DESCRIPTION
**Pull request by keyonghan**
This PR adds linux try config for `deploy_gallery` test.

Related issue: https://github.com/flutter/flutter/issues/66707

### CODE REVIEW

File: config/framework_config.star [hunk 1]
```diff
@@ -609,6 +609,22 @@ def framework_try_config():
             swarming.cache(name = "pub_cache", path = ".pub_cache"),
         ],
     )
+    common.linux_try_builder(
```
Comment (godofredoc) [inline config/framework_config.star:612]: Can we also added to prod?

**Review by godofredoc — APPROVED** [review]

File: config/framework_config.star (hunk 1 above)
Comment (keyonghan) [reply config/framework_config.star:612]: Created this for testing. Once the recipe is validated, I will create a prod config.

Comment (godofredoc) [reply config/framework_config.star:612]: You don't need to create the config to test the recipe you can do:

```led get-builder "luci.flutter.try: Linux docs" > properties.json```

Modify the properties section with the dependencies and secrets required, then do:

```cat properties.json | led edit-recipe-bundle | led launch```

Comment (keyonghan) [reply config/framework_config.star:612]: This is handy! Will try it out.

**Pull request merged**